travis/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+out/host_debug_unopt/ftl_unittests
+out/host_debug_unopt/synchronization_unittests
+out/host_debug_unopt/wtf_unittests
+
+pushd flutter/testing/dart
+pub get
+popd
+
+for TEST_SCRIPT in flutter/testing/dart/*.dart; do
```

Comment:
i recommend having a control test that verifies that if one of these fails, it properly gets noticed. See the framework test.sh for (a paranoidly high number of) examples.